### ISSUE TITLE
INSTABOT_CONFIG_FILE environmental variable ignored

### ISSUE THREAD

**Issue by whocodes** [instabot-py/instabot.py#2314]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x] Bug
- [ ] Feature Request
- [ ] Technical Help
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so, have you tried running the code on your local machine?

#### Operating System? (include version)
- [ ] macOS
- [ ] Windows
- [x] Ubuntu
- [ ] Android (include terminal)

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Exact Python Version?
```
whocodes@whocodes:~$ python3 --version
Python 3.6.8
```

#### Pip Version?
```
whocodes@whocodes:~$ pip3 --version
pip 9.0.1 from /usr/lib/python3/dist-packages (python 3.6)
```

#### Description of your issue
If you launch the script the intended way (`instabot-py` from command line), it ignores the `INSTABOT_CONFIG_FILE` environmental variable.

Console output:
```
whocodes@whocodes:~$ INSTABOT_CONFIG_FILE='instabot.whocodes.yml` instabot-py

INFO:instabot_py.config:Setting configuration from instabot.whocodes.yml : OK

error: Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "instabot_py/__main__.py", line 399, in <module>
    main()
  File "instabot_py/__main__.py", line 390, in main
    configdict = interactive(param, loaded_with_argv=True)
  File "instabot_py/__main__.py", line 351, in interactive
    "Could not find user in settings. Would you like to add now? (yes/no): "
  File "instabot_py/__main__.py", line 34, in ask_question
    with TERM.location(int((TERM.width / 2) - (len(header) / 2)), 1):
TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'
```
(I believe one or two of the errors at the end resulted from me using `Ctrl`+`C` to get out of the console screen)

I literally cannot figure out a way to launch the script with `instabot-py` and have it read a custom configuration file. It ALWAYS opens the interactive console and wants me to setup a new config (which it then saves into the `.ini`).

I am sure that the plugin is making contact with the configuration file, for this reason:
1. I installed the script using `python3 -m pip install instabot.py`
2. I setup and ran the bot using the `.ini` configuration file
3. I setup a `.yml` configuration file and tried launching like above (error like above)
4. I add `Telegram` handler to the `.yml` configuration file and try again
5. I now get error that I am missing `Telegram` handler (yes I know how to fix this)

For this reason I am sure that is making some type of contact with the `.yml` file and just ignoring it.

#### Config

```
---
login : "***"
password : "***"

debug: 1
#Send INFO notification to Telegram channel 
logging.handlers.telegram:
  level: INFO
  class: telegram_handler.TelegramHandler
  token: 937034595:AAEfRsyMeCLuLcJg8AU62qctF3UxspEWQoM
  chat_id: 971135038
logging.loggers.InstaBot.handlers:
  - telegram
  - console

follow_time: 36000
unfollow_per_day: 200
follow_per_day: 400
```

**Comment by whocodes** [comment]
It seems I also get this same issue when traveling to `python3.6/site-packages/instabot_py` and running it with `INSTABOT_CONFIG_FILE='instabot.whocodes.yml' python3 __main__.py`.

**Comment by yurilaaziz** [comment]
https://medium.com/@asker_amine/whats-new-in-the-instagram-bot-0-5-2-fe52154fdc8a

**Comment by loganyc1934** [comment]
No this isn't working. Running `instabot-py --dump > myconfig.yml` leads to

```
INFO:instabot_py.config:Setting configuration from instabot.config.yml : OK
Traceback (most recent call last):
  File "/home/pi/Web/insta/venv/bin/instabot-py", line 8, in <module>
    sys.exit(main())
  File "/home/pi/Web/insta/venv/lib/python3.7/site-packages/instabot_py/__main__.py", line 390, in main
    configdict = interactive(param, loaded_with_argv=True)
  File "/home/pi/Web/insta/venv/lib/python3.7/site-packages/instabot_py/__main__.py", line 351, in interactive
    "Could not find user in settings. Would you like to add now? (yes/no): "
  File "/home/pi/Web/insta/venv/lib/python3.7/site-packages/instabot_py/__main__.py", line 34, in ask_question
    with TERM.location(int((TERM.width / 2) - (len(header) / 2)), 1):
TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'
```

Something is wrong getting the user config. Actually, `instabot-py` with any arguments goes to interactive questions for setting up user config.

https://github.com/instabot-py/instabot.py/issues/2421#issuecomment-548602527

**Comment by mrallanodreman** [comment]
This is running?

**Comment by loganyc1934** [comment]
Ok just got around it by installing from source. This time `instabot-py --version` at least gives me `0.5.9` and not directly to interactive mode.

And now I can run it with `nohup`. Like

```
nohup instabot-py -c myconfig.yml --login xxx --password xxx --ignore-updates &
```

Issue resolved. 

One funny thing is that without `--ignore-updates` it says an older version is newer than current

```
Newer version available: 0.4.10, The current version: 0.5.9
```
